src/annotator/integrations/html-side-by-side.ts

```diff
@@ -228,3 +229,7 @@ export function preserveScrollPosition(
 
   return scrollDelta;
 }
+
+export function isSideBySideMode(mode: unknown): mode is SideBySideMode {
+  return typeof mode === 'string' && ['auto', 'manual'].includes(mode);
```

Comment:
The first clause of the `&&` is redundant, since the second will fail if `mode` isn't one of the strings in the array.

Reply:
Perhaps this is a case in which `includes(mode as string)` makes sense.

Reply:
> Perhaps this is a case in which includes(mode as string) makes sense

👍 - TS's error makes sense here, because passing a value of the "wrong" type to `includes` probably indicates an error most of the time, but we know what we're doing here.